shex/sio-model/undiagnosedShape.shex

```diff
@@ -0,0 +1,40 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#> 
+
+:personShape IRI { 
+  a [ sio:SIO_000498 ] ;
+  sio:SIO_000228 @:role1Shape ;
+  sio:SIO_000228 @:role2Shape
+}
+
+:role1Shape IRI {
+  a [ obo:OBI_0000093 ] ;
+  sio:SIO_000356 @:process1Shape
+}
+
+:process1Shape IRI {
+  a [ obo:NCIT_C15709 ] ;
+  sio:SIO_000229  @:variantShape
+}
+
+:variantShape IRI {
+  a [ obo:VariO_0138 ] ;
+  rdfs:label xsd:string
+}
+
+:role2Shape IRI {
+  a [ obo:OBI_0000093 ] ;
+  sio:SIO_000356 @:process2Shape
+}
+
+:process2Shape IRI {
+  a [ obo:NCIT_C16205 ] ;
+  sio:SIO_000229  @:phenotypeShape
+}
+
+:phenotypeShape IRI {
+  a [ obo:HP_0001249 ] 
```

Comment:
This HPO uri is used as an example in the examle ttl file. Can you replace it with regular expression?

Reply:
Sure, the use of HPO ontology should be included at this Regular expression, or this shape could have any different type of OBO terms?

Reply:
Regular expression of HPO is enough